src/game/default/g_types.h

```diff
@@ -515,6 +523,8 @@ typedef struct {
 
 		uint16_t weapon_no_ammo;
 		uint16_t weapon_switch;
+		
+		uint16_t countdown[11];
```

Comment:
wait does lengthof() actually work? I figured I had to divide the sizeof of the array by the sizeof one of the elements...